.github/workflows/update-infrastructure-docker.yaml

```diff
@@ -0,0 +1,31 @@
+name: Update infrastructure-docker submodule
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  schema:
+    name: Update infrastructure-docker submodule
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: "misarch/infrastructure-docker"
+          submodules: true
+      - name: Update submodule
+        run: |
+          cd experiment-config-sidecar
+          git checkout ${{ github.sha }}
+      - uses: peter-evans/create-pull-request@v6
+        with:
+          commit-message: Update experiment-config-sidecar schema
```

Comment:
Is there experiment-config-sidecar schema?

Reply:
this is a copy paste error which can be found in most services
maybe I should fix this at some point in time...
here fixed with a89035f4e4739ce946a05c84a5867919ec4fcb67